src/hooks/validation.ts

```diff
@@ -57,7 +57,9 @@ const resourceSearchValidators: QueryParamValidators<ResourceSearchRequest> = {
   id:                 numbers,
   free:               firstBoolean,
   learning_format:    withinEnum(Object.values(LearningFormatEnum)),
-  certification_type: withinEnum(Object.values(CertificationTypeEnum))
+  certification_type: withinEnum(Object.values(CertificationTypeEnum)),
+  // @ts-ignore
+  resource_category: identity
```

Comment:
This should be changed to `withinEnum(Object.values(ResourceCategoryEnum))` once https://github.com/mitodl/mit-open/pull/1188 is released on production and @mitodl/open-api-axios can be updated